Tasks/InstallSSHKey/installsshkey-util.ts

```diff
@@ -24,18 +24,9 @@ function execSshAddPassphraseSync(tool, args, passphrase):Q.Promise<boolean> {
     var defer = Q.defer<boolean>();
     let success = true;
 
-    let options:trm.IExecOptions = <trm.IExecOptions> {
-        cwd: process.cwd(),
-        env: process.env,
-        silent: false,
-        failOnStdErr: false,
-        ignoreReturnCode: true,
-        windowsVerbatimArguments: false,
-        outStream:<stream.Writable>process.stdout,
-        errStream:<stream.Writable>process.stderr
-    };
-
-    let cp = child.spawn(tool, args, {});
+    let cp = child.spawn(tool, args, {
```

Comment:
To close the loop on this: the process should end immediately, so the 'detached' option shouldn't affect anything adversely, and it adds another code-path, which is why I didn't add OS specific logic.  So, I don't feel this matters either way.  (FWIW, I have tested on Windows, Linux, and macOS.)

In any case, I don't have strong feelings about this, and I don't know which way carries more risk.  If you think one way is superior to the other, let me know and I'll change it.